src/test/java/io/xstefank/wildfly/bot/PROpenedTest.java

```diff
@@ -421,6 +424,61 @@ void testPullRequestFormatTitleCheckOnOpen() throws IOException {
             });
     }
 
+    @Test
+    void testTitleBodyCheckTitleRegexOnOpen() throws IOException {
```

Comment:
There is a different regex check on EDIT? If not, I would remove the "onOpen" do prevent confusion